samples/Kconfig

```diff
@@ -0,0 +1,8 @@
+# General configuration options for samples
+
+# Copyright (c) 2025 The Zephyr Project Contributors
+# SPDX-License-Identifier: Apache-2.0
+
+config SYSTEM_WORKQUEUE_STACK_MIN_SIZE_SAMPLE
+	int "System workqueue stack size for a sample application"
+	default 1024
```

Comment:
Note that as this file is always sourced, this will be a minimum in general for any app (test/sample or anything else).
(it seems off to me to create a new kconfig file for this at this level)
Maybe instead either:
* there could be some more generic SYSTEM_WORKQUEUE_STACK_MIN_SIZE_APP defined in kernel/Kconfig, 
* or CONFIG_SYSTEM_WORKQUEUE_STACK_SIZE could default to this 1024 (To me it seems that defaulting CONFIG_SYSTEM_WORKQUEUE_STACK_SIZE to 0 is wrong, as with 0 nothing can work)

?